minetest/minetest
# Fix rendering of translucent items in inventory & when wielded

Closes #5461. Ready for review.

## How to test

Look at the three marbles, both in the inventory and as wield items:

* The opaque marble should be opaque.
* The clipped marble should be clipped.
* The blended marble should be blended.

Also look at the alpha test node with alpha = 64. It should be visible, both in your inventory and when wielded. It should look like this:

![Screenshot](https://github.com/minetest/minetest/assets/34514239/dc509d57-685b-4a66-ad77-a04411ef4934)

Cycle through the devtest items and verify that they look right; also look at some simple wield items, like pickaxes, to make sure that there are no regressions.

Furthermore, enable `inventory_items_animations` and check that the spinny items look as expected.

Relevant documentation: https://irrlicht.sourceforge.io/docu/namespaceirr_1_1video.html#ac8e9b6c66f7cebabd1a6d30cbc5430f1ac08aa3715ad41281472202107a81f736
